src/channel_internal.c

```diff
@@ -457,13 +404,13 @@ IMPLEMENT_MOCKABLE_FUNCTION(, CHANNEL_RESULT, channel_internal_push, THANDLE(CHA
                 result = CHANNEL_RESULT_OK;
             }
         }
-        /*Codes_SRS_CHANNEL_INTERNAL_43_124: [ If the first operation in the list of pending operations contains a non-NULL pull_callback: ]*/
-        else if (CONTAINING_RECORD(channel_internal_ptr->op_list.Flink, CHANNEL_OP, anchor)->pull_callback != NULL)
+        /*Codes_SRS_CHANNEL_INTERNAL_43_124: [ Otherwise (the first operation in the list of pending operations contains a non-NULL pull_callback): ]*/
+        else
         {
             if (dequeue_operation(channel_internal, out_op_push, NULL, NULL, NULL, correlation_id, push_callback, push_context, data) != 0)
             {
                 /*Codes_SRS_CHANNEL_INTERNAL_43_041: [ If there are any failures, channel_internal_push shall fail and return CHANNEL_RESULT_ERROR. ]*/
-                LogError("Failure in dequeue_operation(channel_internal_ptr, out_op_push, NULL, NULL, NULL, correlation_id, push_callback, push_context, data)");
+                LogError("Failure in dequeue_operation(channel_internal = %p, out_op_push = %p, NULL, NULL, NULL, correlation_id = %" PRI_RC_STRING ", push_callback = %p, push_context = %p, data = %p)", channel_internal, out_op_push, RC_STRING_VALUE_OR_NULL(correlation_id), push_callback, push_context, data);
```

Comment:
> =  [](http://example.com/codeflow?start=164&length=3)

Nitpick: usually when searching in logs it is more convenient to search for "partitionId=" then "partitionId = ".

Now it is true that we're not fully consistent about this throughout the code base, but also it is true that most of the logs are without spaces.

So I would suggest ditching the spaces. /consider.